cmd/lncli/commands.go

```diff
@@ -1401,6 +1401,101 @@ func create(ctx *cli.Context) error {
 	client, cleanUp := getWalletUnlockerClient(ctx)
 	defer cleanUp()
 
+	// We'll check to see if the user provided any static channel backups (SCB),
+	// if so, we will warn the user that SCB recovery closes all open channels
+	// and ask them to confirm their intention.
+	//
+	// If the user agrees, we'll add the SCB recovery onto the final init wallet
+	// request.
+
+	var (
+		chanBackups *lnrpc.ChanBackupSnapshot
+
+		// We use var restoreSCB to track if we will be including an SCB
+		// recovery in the init wallet request.
+		//
+		// By default, we do not include the SCB recovery.
+		restoreSCB = false
+	)
+
+	// parseChanBackups returns an errMissingBackup error if there is no
+	// (optional) SCB backup file in the command.
+	//
+	// We can therefore ignore the errMissingChanBackup error as it's an
+	// optional field.
+	//
+	// Try to load an SCB file.
+	backups, err := parseChanBackups(ctx)
+
+	// Evaluate result (err) from parseChanBackups.
+	switch {
+	case err == errMissingChanBackup:
+		// SCB recovery is an optional parameter of the create command.
```

Comment:
style nit: we usually put switch comments above the `case` instead of below. This avoids some indentation  :)

Reply:
Fixing